examples/hosts/app-integration/external-data/src/model/containerCode.ts

```diff
@@ -62,11 +62,12 @@ export class TaskListContainerRuntimeFactory extends ModelContainerRuntimeFactor
         );
         // Register listener only once the model is fully loaded and ready
         runtime.on("signal", (message) => {
-            if (message.type === SignalType.ExternalDataChanged) {
+        // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+        if ('content' in message && 'type' in message.content && message.content.type === SignalType.ExternalDataChanged) {
```

Comment:
```suggestion
        if (message?.content?.type === SignalType.ExternalDataChanged) {
```
Can it be done this way?